lumen/ai/agents.py

```diff
@@ -695,8 +695,8 @@ async def respond(
             table=table,
         )
         if join_required:
-            # Remove source prefixes message, e.g. //<source>//<table>
-            messages[-1]["content"] = re.sub(r"//[^/]+//", "", messages[-1]["content"])
+            # Remove source prefixes message, e.g. SOURCE_TABLE_SEPARATOR<source>SOURCE_TABLE_SEPARATOR<table>
+            messages[-1]["content"] = re.sub(r".*?<->", "", messages[-1]["content"])
```

Comment:
Maybe this can be 
`r".*?" + SOURCE_TABLE_SEPARATOR` (if it works)

Reply:
good catch

Reply:
`rf".*?{SOURCE_TABLE_SEPARATOR}"` works